temporal-sdk/src/main/java/io/temporal/workflow/ChildWorkflowOptions.java

```diff
@@ -303,6 +307,31 @@ public Builder setVersioningIntent(VersioningIntent versioningIntent) {
       return this;
     }
 
+    /**
+     * Single-line fixed summary for this workflow execution that will appear in UI/CLI. This can be
+     * in single-line Temporal Markdown format.
+     *
+     * <p>Default is none/empty.
+     */
+    @Experimental
+    public Builder setStaticSummary(String staticSummary) {
```

Comment:
This is not gonna work until https://github.com/temporalio/temporal/pull/6446 is in a released server version, unsure if you want to wait to expose until then. I do see the test is skipped currently for real servers.

Reply:
Yes this should be in `1.25.0` and the tests are skipped for the test server, not the real server

Reply:
Hrmm, child workflows may not have made the initial version (they weren't initially wired up). @gow - can you confirm whether or not the child workflow user metadata propagation is in a tagged server version?

Reply:
https://github.com/temporalio/temporal/commit/dd0c6ca01265298483408636da4d52da2b8b327a is part of [release/v1.25.x](https://github.com/temporalio/temporal/compare/release/v1.25.x) and a parent of https://github.com/temporalio/temporal/commit/be848c89192af6c5f9f01477ab2320b6a40d4b67

Reply:
Yes. It's included in 1.25 release - https://github.com/temporalio/temporal/compare/v1.25.0...main